src/backupNg/__snapshots__/backupNg.spec.js.snap

```diff
@@ -9,7 +9,7 @@ Object {
   "type": "backup",
   "userId": Any<String>,
   "vms": Object {
-    "id": "7146cbc8-d817-850f-92e4-ba39e40e701f",
```

Comment:
I should do a PR to remove the exact value of the `vmId` in the snapshots, check only the type & check the value in the code. What do you think ? This should avoid changing the snapshots each time we change the `vmId`.

Reply:
I removed this useless change. Yes, changing the config should not break tests.